sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/complexTypeCreator.scala

```diff
@@ -234,7 +259,16 @@ case class CreateStruct(children: Seq[Expression]) extends Expression {
  */
 // scalastyle:off line.size.limit
 @ExpressionDescription(
-  usage = "_FUNC_(name1, val1, name2, val2, ...) - Creates a struct with the given field names and values.")
+  usage = "_FUNC_(name1, val1, name2, val2, ...) - Creates a struct with the given field names and values.",
+  extended = """
+    Arguments:
+      name - a string expression literal that represents the field name.
+      val - an expression of any type.
+
+    Examples:
+      > SELECT _FUNC_("a", 1, "b", 2, "c", 3);
```

Comment:
Could you follow the consistent way to specify the string literal in the example? Most of them are using single quotes